### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by somersbmatthews**

### CODE REVIEW

**Comment by somersbmatthews** [comment]
#9

File: README.md [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+COSI repos:
+
+|Repo | Images | 
+|----------------|-----------|
+| [Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec) | N/A  |
+| [API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) | N/A |
+| [Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) | [cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) |
+| [Provisioner Sidecar](https://github.com/kubernetes-sigs/container-object-storage-interface-provisioner-sidecar) | cosi-provisioner TODO |
+| [CSI Adapter](https://github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter) | cosi-node-adapter TODO |
+\
```
Comment (sajanjswl) [inline README.md:10]: Can you please fix this? And also squash all the commits. Other than that it looks good to me.

Comment (somersbmatthews) [reply README.md:10]: I asked you about this in https://github.com/kubernetes-sigs/container-object-storage-interface-spec/pull/15 , will git rebase first thing tomorrow.

**Comment by somersbmatthews** [comment]
@sajanjswl @xing-yang squashed commits and removed table.

**Comment by somersbmatthews** [comment]
@sajanjswl @xing-yang squashed commits and removed table.

**Review by sajanjswl — APPROVED** [review]

**Pull request merged**